Cannot link with stdint on Linux when using Uint128_val when HAVE_UINT128

```shell
ocamlfind ocamlopt -linkpkg -package stdint foo.ml -o foo.exe
/usr/bin/ld: src/libbar.a(bar.o): in function `baz':
bar.c:123: undefined reference to `get_uint128'
collect2: error: ld returned 1 exit status
File "caml_startup", line 1:
Error: Error during linking
```

--- a/CHANGES.md
+++ b/CHANGES.md
@@ -1,3 +1,9 @@
+# Unreleased
+
+## Fixes:
+
+* Fix undefined reference to `get_uint128` (#62)
+
 # 0.7.0 (28/10/2020)
 
 ## Fixes:

--- a/lib/int128_stubs.c
+++ b/lib/int128_stubs.c
@@ -15,7 +15,10 @@
 #include "uint128.h"
 #include "int128.h"
 
-#ifndef HAVE_INT128
+#ifdef HAVE_INT128
+extern inline __int128_t get_int128(value);
+#else
+
 void divmod128(int128 *d, int128 *modulus, int128 *quotient, int128 *rem);
 
 static inline int32_t compare(int128 *x, int128 *y) {

--- a/lib/uint128_stubs.c
+++ b/lib/uint128_stubs.c
@@ -21,7 +21,10 @@
 #include "uint64.h"
 #include "uint128.h"
 
-#ifndef HAVE_UINT128
+#ifdef HAVE_UINT128
+extern inline __uint128_t get_uint128(value);
+#else
+
 static inline int compare(uint128 *x, uint128 *y) {
   assert(x);
   assert(y);

